### ISSUE TITLE
Requests not captured after updating to 4.5.5

### ISSUE THREAD

**Issue by iwanbolzern** [wkeeling/selenium-wire#452]
After updating from 4.5.4 --> 4.5.5 the requests are not captured anymore. 

My setup:
Python 3.8.9
Dependencies:
async-generator    1.10      Async generators and context managers for Python 3.5+
atomicwrites       1.4.0     Atomic file writes.
attrs              21.2.0    Classes Without Boilerplate
blinker            1.4       Fast, simple object-to-object and broadcast signaling
boto3              1.20.11   The AWS SDK for Python
botocore           1.23.11   Low-level, data-driven core of boto 3.
brotli             1.0.9     Python bindings for the Brotli compression library
certifi            2021.10.8 Python package for providing Mozilla's CA Bundle.
cffi               1.15.0    Foreign Function Interface for Python calling C code.
charset-normalizer 2.0.7     The Real First Universal Charset Detector. Open, modern and actively maintained alternative to Chardet.
click              8.0.3     Composable command line interface toolkit
colorama           0.4.4     Cross-platform colored terminal text.
configparser       5.1.0     Updated configparser from Python 3.8 for Python 2.6+.
crayons            0.4.0     TextUI colors for Python.
cryptography       36.0.0    cryptography is a package which provides cryptographic recipes and primitives to Python developers.
h11                0.12.0    A pure-Python, bring-your-own-I/O implementation of HTTP/1.1
h2                 4.1.0     HTTP/2 State-Machine based protocol implementation
hpack              4.0.0     Pure-Python HPACK header compression
hyperframe         6.0.1     HTTP/2 framing layer for Python
idna               3.3       Internationalized Domain Names in Applications (IDNA)
ifaddr             0.1.7     Cross-platform network interface and IP address enumeration library
importlib-metadata 4.8.2     Read metadata from Python packages
iniconfig          1.1.1     iniconfig: brain-dead simple config-ini parsing
jmespath           0.10.0    JSON Matching Expressions
kaitaistruct       0.9       Kaitai Struct declarative parser generator for binary data: runtime library for Python
keyring            23.2.1    Store and access your passwords safely.
outcome            1.1.0     Capture the outcome of Python function calls.
packaging          21.3      Core utilities for Python packages
pluggy             1.0.0     plugin and hook calling mechanisms for python
psutil             5.8.0     Cross-platform lib for process and system monitoring in Python.
py                 1.11.0    library with cross-python path, ini-parsing, io, code, log facilities
pyasn1             0.4.8     ASN.1 types and codecs
pycparser          2.21      C parser in Python
pydivert           2.1.0     Python binding to windivert driver
pyopenssl          21.0.0    Python wrapper module around the OpenSSL library
pyparsing          3.0.6     Python parsing module
pysocks            1.7.1     A Python SOCKS client module. See https://github.com/Anorov/PySocks for more information.
pytest             6.2.5     pytest: simple powerful testing with Python
python-dateutil    2.8.2     Extensions to the standard Python datetime module
pywin32-ctypes     0.2.0
requests           2.26.0    Python HTTP for Humans.
s3transfer         0.5.0     An Amazon S3 Transfer Manager
selenium           4.1.0
selenium-wire      4.5.5     Extends Selenium to give you the ability to inspect requests made by the browser.
six                1.16.0    Python 2 and 3 compatibility utilities
sniffio            1.2.0     Sniff out which async library your code is running under
sortedcontainers   2.4.0     Sorted Containers -- Sorted List, Sorted Dict, Sorted Set
toml               0.10.2    Python Library for Tom's Obvious, Minimal Language
trio               0.19.0    A friendly Python library for async concurrency and I/O
trio-websocket     0.9.2     WebSocket library for Trio
typer              0.4.0     Typer, build great CLIs. Easy to code. Based on Python type hints.
urllib3            1.26.7    HTTP library with thread-safe connection pooling, file post, and more.
webdriver-manager  3.5.2     Library provides the way to automatically manage drivers for different browsers
wsproto            1.0.0     WebSockets state-machine based protocol implementation
xmltodict          0.12.0    Makes working with XML feel like you are working with JSON
zipp               3.6.0     Backport of pathlib-compatible object wrapper for zip files
zstandard          0.16.0    Zstandard bindings for Python

**Comment by wkeeling** [comment]
Thanks for raising this. 

Can you also let me know the following:

* Browser + version
* Driver version (chromedriver or geckodriver)
* OS + version

Thanks!

**Comment by iwanbolzern** [comment]
Thanks you for looking into this. The versions are:

Browser: Chrome Version 96.0.4664.45
Driver: https://chromedriver.storage.googleapis.com/LATEST_RELEASE ('96.0.4664') managed via webdriver_manager
OS: Windows 20H2 19042.1288

**Comment by wkeeling** [comment]
Thanks. I've not been able to reproduce the problem with: Selenium Wire 4.5.5, Selenium 4.1.0, Chrome 96.0.4664.45, chromedriver 96.0.4664.45 on Windows 10. The requests are captured as expected.

I wonder whether there is something else going on in your environment that may be causing the problem. Are you able to try creating a fresh virtual environment, install Selenium Wire 4.5.5 into that, and then retry running a test?

**Comment by FranciscoPalomares** [comment]
I have the same problem:
Versión chromedriver: '96.0.4664.45'

**Comment by FranciscoPalomares** [comment]
With  selenium-wire==4.5.4 works again

**Comment by iwanbolzern** [comment]
> Thanks. I've not been able to reproduce the problem with: Selenium Wire 4.5.5, Selenium 4.1.0, Chrome 96.0.4664.45, chromedriver 96.0.4664.45 on Windows 10. The requests are captured as expected.
> 
> I wonder whether there is something else going on in your environment that may be causing the problem. Are you able to try creating a fresh virtual environment, install Selenium Wire 4.5.5 into that, and then retry running a test?
> 
> Also I should just check - are you passing any `seleniumwire_options` when you create the webdriver?

Thanks for trying to reproduce!

I can allocate some time in the afternoon to checkout the repo and try to run the tests.

I do not pass any further `seleniumwire_options`. I only pass `window-size` and `log-level` as chrome options.

The behavior is exactly the same as @FranciscoPalomares describes. Therefore, I assume the problem must be somewhere in the implementation and not in the setup.

**Comment by wkeeling** [comment]
Thanks @FranciscoPalomares . Can I ask which version of Selenium you are running?

**Comment by FranciscoPalomares** [comment]
@wkeeling selenium==4.1.0

**Comment by wkeeling** [comment]
Thanks @FranciscoPalomares and @iwanbolzern 

Given that we seem to be using the same software versions and you are experiencing the problem whilst I cannot reproduce, would you be able to post a minimal code example that reproduces - including all arguments you're passing to the `ChromeDriver()`? I'll then have another try using that Thanks!

**Comment by FranciscoPalomares** [comment]
try with:
```
  from webdriver_manager.chrome import ChromeDriverManager
  driver = webdriver.Chrome(ChromeDriverManager().install(), seleniumwire_options=options, chrome_options=options_chrome)
```

**Comment by wkeeling** [comment]
It seems that this issue is caused by passing `ChromeOptions` to the driver using the now deprecated `chrome_options` argument. This overwrites Selenium Wire's own configuration. I'll fix this in the next release, but for now if you use the `options` argument it should resolve the problem. 

```python
driver = webdriver.Chrome(
    seleniumwire_options=options,
    options=options_chrome  # Use the `options` argument for ChromeOptions
)
```

**Comment by iwanbolzern** [comment]
@wkeeling Thanks a lot for resolving this so quickly. By using `options` instead of `chrome_options` it's working perfectly fine.

Have a great weekend!

**Comment by wkeeling** [comment]
Now fixed in v4.5.6
